<ui version="4.0" >
 <class>StartupDialog</class>
 <widget class="QDialog" name="StartupDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>264</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Select DVRCP System</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox" >
   <property name="geometry" >
    <rect>
     <x>270</x>
     <y>223</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons" >
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2" >
   <property name="geometry" >
    <rect>
     <x>30</x>
     <y>30</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text" >
    <string>Select an existing system configuration folder to initialize the DVRCP. Leave it blank to create a new system configuration.</string>
   </property>
   <property name="alignment" >
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap" >
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="uiGrpStartupSystem" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>431</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title" >
    <string>Choose a firmware and system configuration:</string>
   </property>
   <widget class="QLabel" name="uiLableSystemFolder" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>61</y>
      <width>102</width>
      <height>21</height>
     </rect>
    </property>
    <property name="whatsThis" >
     <string/>
    </property>
    <property name="text" >
     <string>System folder:</string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="uiBrowseSystemFolder" >
    <property name="geometry" >
     <rect>
      <x>380</x>
      <y>61</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="uiLabelStartupFirmware" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>165</y>
      <width>104</width>
      <height>17</height>
     </rect>
    </property>
    <property name="whatsThis" >
     <string/>
    </property>
    <property name="text" >
     <string>Startup Firmware:</string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="uiBrowseStartupFirmware" >
    <property name="geometry" >
     <rect>
      <x>380</x>
      <y>165</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>100</y>
      <width>391</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text" >
     <string>Select a romable firmware file to load on to the DVR board for this session. Leave it blank to use the exsiting firmware flashed on the DVR board.</string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap" >
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>20</y>
      <width>381</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text" >
     <string>Select an existing system configuration folder to initialize the DVRCP. Leave it blank to create a new system configuration.</string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap" >
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="uiCombboxSystemFolder" >
    <property name="geometry" >
     <rect>
      <x>130</x>
      <y>60</y>
      <width>241</width>
      <height>22</height>
     </rect>
    </property>
    <property name="whatsThis" >
     <string>Select an existing system configuration folder to initialize the DVRCP. Leave it blank to create a new system configuration.</string>
    </property>
    <property name="editable" >
     <bool>true</bool>
    </property>
    <property name="maxVisibleItems" >
     <number>4</number>
    </property>
    <property name="insertPolicy" >
     <enum>QComboBox::InsertAtTop</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="uiCombboxStartupFirmware" >
    <property name="geometry" >
     <rect>
      <x>130</x>
      <y>164</y>
      <width>241</width>
      <height>22</height>
     </rect>
    </property>
    <property name="whatsThis" >
     <string>Select a romable firmware file to load on to the DVR board for this session. Leave it blank to use the exsiting firmware flashed on the DVR board.</string>
    </property>
    <property name="editable" >
     <bool>true</bool>
    </property>
    <property name="maxVisibleItems" >
     <number>4</number>
    </property>
    <property name="insertPolicy" >
     <enum>QComboBox::InsertAtTop</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>uiCombboxSystemFolder</tabstop>
  <tabstop>uiBrowseSystemFolder</tabstop>
  <tabstop>uiCombboxStartupFirmware</tabstop>
  <tabstop>uiBrowseStartupFirmware</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>StartupDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>StartupDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
